### PR TITLE
cdc: add advice for how to use kafka sink in ticdc

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)(v5.3.1)
- [x] v5.2 (TiDB 5.2 versions) (v5.2.3)
- [x] v5.1 (TiDB 5.1 versions) (v5.1.4)
- [x] v5.0 (TiDB 5.0 versions) (v5.0.6)
- [x] v4.0 (TiDB 4.0 versions)（v4.0.16)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -215,6 +215,15 @@ URI 中可配置的的参数如下：
...
 | `sasl-mechanism` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的名称（可选） |
 
+用户使用方式建议:
+
+* TiCDC 推荐用户自行创建 Kafka Topic，至少需要知道 Topic 的 `max.message.bytes` 参数和 partition 的数量。在创建 changefeed 的时候，分别对应设置 `max-message-bytes` 和 `partition-num` 参数。
+* 如果用户在创建 changefeed 时，使用了尚未存在的 Topic，那么 TiCDC 会尝试使用 `partition-num` 和 `replication-factor` 参数自行创建 Topic。建议用户明确指定这两个参数。
+
+参数详细说明:
+
+* v4.0.16, `max-message-bytes` 默认值从 512MB 调整为 1MB, `partition-num` 默认值从 4 调整为 3。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:225]: 在 v5.0、v5.1、v5.2 中各是从哪个版本调整的呢？

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -202,11 +202,11 @@ URI 中可配置的的参数如下：
 | :------------------ | :------------------------------------------------------------ |
 | `127.0.0.1`          | 下游 Kafka 对外提供服务的 IP                                 |
 | `9092`               | 下游 Kafka 的连接端口                                          |
-| `topic-name`           | 变量，使用的 Kafka topic 名字                                      |
-| `kafka-version`      | 下游 Kafka 版本号。可选，默认值 `2.4.0`，目前支持的最低版本为 `0.11.0.2`，最高版本为 `2.7.0`。该值需要与下游 Kafka 的实际版本保持一致。 |
-| `kafka-client-id`    | 指定同步任务的 Kafka 客户端的 ID。可选，默认值为 `TiCDC_sarama_producer_同步任务的 ID`。 |
-| `partition-num`     | 下游 Kafka partition 数量。可选，不能大于实际 partition 数量。如果不填，会自动获取 partition 数量。 |
-| `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `64MB`） |
+| `cdc-test`           | 使用的 Kafka topic 名字                                      |
```
Comment (hi-rustin) [inline ticdc/manage-ticdc.md:205]: 这是一个 typo 吗？

**Review by leoppro — APPROVED** [review]
LGTM

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -202,11 +202,11 @@ URI 中可配置的的参数如下：
...
+| `kafka-version`      | 下游 Kafka 版本号（可选，默认值 `2.4.0`，目前支持的最低版本为 `0.11.0.2`，最高版本为 `2.7.0`。该值需要与下游 Kafka 的实际版本保持一致） |
+| `kafka-client-id`    | 指定同步任务的 Kafka 客户端的 ID（可选，默认值为 `TiCDC_sarama_producer_同步任务的 ID`） |
+| `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量, 否则创建同步任务失败, 默认值 `3`)|
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:208]:
```suggestion
| `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）|
```

File: ticdc/manage-ticdc.md (hunk 2 above)
Comment (TomShawn) [reply ticdc/manage-ticdc.md:205]: 这里其实是配置项变更，从 v4.0.16 起 由 cdc-test 表示“使用的 Kafka topic 名字”，而之前的 topic-name 被弃用？

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -217,6 +217,15 @@ URI 中可配置的的参数如下：
 | `sasl-password` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的密码（可选） |
 | `sasl-mechanism` | 连接下游 Kafka 实例所需的 SASL/PLAIN 或 SASL/SCRAM 验证的名称（可选） |
 
+用户使用方式建议:
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:220]:
```suggestion
最佳实践：
```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -217,6 +217,15 @@ URI 中可配置的的参数如下：
...
+
+* TiCDC 推荐用户自行创建 Kafka Topic，至少需要知道 Topic 的 `max.message.bytes` 参数和 partition 的数量。在创建 changefeed 的时候，分别对应设置 `max-message-bytes` 和 `partition-num` 参数。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:222]:
```suggestion
* 在 TiCDC Sink URI 中配置 kafka 时，推荐自行创建 Kafka Topic。你至少需要设置该 Topic 每次向 Kafka broker 发送消息的最大数据量和下游 Kafka partition 的数量。在创建 changefeed 的时候，这两项设置分别对应 `max-message-bytes` 和 `partition-num` 参数。
```

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -217,6 +217,15 @@ URI 中可配置的的参数如下：
...
+* 如果用户在创建 changefeed 时，使用了尚未存在的 Topic，那么 TiCDC 会尝试使用 `partition-num` 和 `replication-factor` 参数自行创建 Topic。建议用户明确指定这两个参数。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:223]:
```suggestion
* 如果你在创建 changefeed 时，使用了尚未存在的 Topic，那么 TiCDC 会尝试使用 `partition-num` 和 `replication-factor` 参数自行创建 Topic。建议明确指定这两个参数。
```

File: ticdc/manage-ticdc.md [hunk 7]
```diff
@@ -217,6 +217,15 @@ URI 中可配置的的参数如下：
...
+
+参数详细说明:
+
+* v4.0.16, `max-message-bytes` 默认值从 512MB 调整为 1MB, `partition-num` 默认值从 4 调整为 3。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:227]: 这句话可以在 master 版本中去掉，因为 master 永远是最新的，没有历史包袱。在 release-4.0 中加回这句话即可。

Comment (3AceShowHand) [reply ticdc/manage-ticdc.md:227]: 之后再 cherry-pick 到 4.0 的 PR 上做出修改即可吧？

Comment (TomShawn) [reply ticdc/manage-ticdc.md:227]: 是的

File: ticdc/manage-ticdc.md (hunk 2 above)
Comment (hi-rustin) [reply ticdc/manage-ticdc.md:205]: 看上去我们上面的 URI 里面还是 topic-name，所以我觉得是个 typo。

**Review by hfxsd — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
Follow-up task: This PR should also cherry-pick to v5.0, v5.1, v5.2 in the future at specific patch releases.

**Pull request merged**